plugins/source/aws/codegen/recipes/config.go

```diff
@@ -60,6 +60,25 @@ func ConfigResources() []*Resource {
 					},
 				}...),
 		},
+		{
+			SubService: "config_rules",
+			Struct:     new(types.ConfigRule),
+			ExtraColumns: append(defaultRegionalColumns,
```

Comment:
I think `ConfigRuleArn` needs to be skipped here so that it's not a duplicate of `arn`

Reply:
fixed